doc/userguide/rules/flow-keywords.rst

```diff
@@ -143,7 +143,12 @@ Define a var (not required), or check that one is set or not set.
 ::
 
     flowint: name, < +,-,=,>,<,>=,<=,==, != >, value;
-    flowint: name, (isset|isnotset);
+    flowint: name, (isset|notset);
+
+.. note::
+
+    It's important to observe that while similar keywords use ``isnotset``,
```

Comment:
The proper fix here is to add isnotset support, while keeping the `notset` for compatibility reasons

Reply:
Yes, this seems trivial enough. But not getting to it, should not prevent correct documentation :)